test/yamlAST.test.ts

```diff
@@ -0,0 +1,108 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { isNodesEqual, newMap, newMapping, newScalar, newSeq } from '../src';
+import { expect } from 'chai';
+
+suite('YAML AST', () => {
+  suite('AST Node Equality', () => {
+
+    suite('Scalar Nodes', () => {
+      test('Nodes with different kind not equal', () => {
+        const a = newScalar('a');
+        const b = newSeq();
+        const result = isNodesEqual(a, b);
+        expect(result).is.false;
+      });
+  
+      test('Scalar Nodes should be equal', () => {
+        const a = newScalar('a');
+        const b = newScalar('a');
+        const result = isNodesEqual(a, b);
+        expect(result).is.true;
+      });
+  
+      test('Scalar Nodes should be not equal', () => {
```

Comment:
The text for this should be more clear. It is the opposite of the one above. Reading just the test description is confusing. `Nodes should be equal` `Nodes should not be equal`
Without the condition in the description this is hard to follow. 
May I suggest something like `test('should return not equal when the Scalar Nodes do not equal each other', () => {`